js/notify/electron-notify.js

```diff
@@ -208,12 +227,32 @@ function calcDimensions() {
 function setupConfig() {
     closeAll();
 
-    // Use primary display only
-    let display = electron.screen.getPrimaryDisplay();
+    // This feature only applies to windows
+    if (!isMac) {
+        let screens = electron.screen.getAllDisplays();
+        if (screens && screens.length >= 0) {
+            externalDisplay = screens.find((screen) => {
+                let screenId = screen.id.toString();
+                return ((screen.bounds.x !== 0 || screen.bounds.y !== 0) && screenId === notfDisplay);
```

Comment:
Because we need only the external display & not primary display.

Please look at my below comment for more info.